src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -14,7 +14,8 @@ function init() {
     return;
   }
 
-  var mobileCarousel = new MobileCarousel();
+  var breakpointPx = 599;
```

Comment:
I kept it as a variable to self-document it a bit better since it can be hard to tell what the straight-up number is setting. Could be a code comment too. Fixed in https://github.com/cfpb/cfgov-refresh/commit/9c28a42a6d246fac28563104045a6156dfe8725c